04_health/neonatal_health_code_2022.do

```diff
@@ -0,0 +1,701 @@
+************************************************************************
+* Ancestor program: $gitfolder/neonatal_health_code_2018.do 				   
+* Original data: all_births_by_county.txt, lbw_births_by_county.txt, nomiss_bw_by_county.txt available in $gitfolder/04_health/data      				   
+* Description: Program to create gates mobility metrics on neonatal health  
+* Authors: Emily M. Johnston, Julia Long 												   
+* Date: September 2, 2022	
+* (1)  download data from CDC WONDER											
+* (2)  import, clean, and merge CDC WONDER files					   
+* (3)  intermediate file data cleaning										   
+* (4)  use crosswalk to add missing counties to data file		
+* (5)  assign "unidentified county" values to counties with missing values
+* (6)  create neonatal health share lowbirthweight metric
+* (7) assess data quality
+* (8) construct 95 percent confidence intervals
+* (9) final file cleaning and export to csv file	
+* more steps that start with taking the final county file
+	*sum the numerators/denoms 
+	*use the same confidence interval formula. 
+****************************************************************************
+
+
+/*
+
+Description: Program to Update Gates Mobility Metrics on Neonatal Health
+Author: Jay Carter
+Date: Feb 21, 2024
+
+Data from CDC Wonder - Instructions in Readme
+
+*/
+clear all 
+pause on 
+
+global gitfolder "C:\Users\jcarter\Documents\git_repos\mobility-from-poverty\"	// update path as necessary to the local mobility metrics repository folder
+global health "${gitfolder}04_health\"
+global health_data "${health}data\"
+global health_data_final "${health}final_data\"
+global geo_xwalk "${gitfolder}geographic-crosswalks\data\"
+
+global sub = "nhblack hisp nhother nhwhite"
+global ed_sub "lessthanhs hsgrad somecollege collegedegrees"
+global data_types = "all raceth momed"
+
+cd "${gitfolder}"
+
+
+local y2 = 22
+local y4 = 2000 + `y2'
+
+//# *(1) download data from CDC WONDER
+
+* access instructions for completing the CDC WONDER data query in README	
+* data query results are saved as lbw_births_by_county.txt, nomiss_bw_by_county.txt available in $gitfolder/04_health/data 
+
+//# *(2) open, clean, and merge CDC WONDER files
+
+//# open and clean data: all births
+//# All Low Birthweight Births
+import delimited using "${health_data}lbw_births_by_county_`y2'.txt", clear
```

Comment:
Code repeats from here through line 83 -- could place the file names in a macro and creating a loop that pulls those file names in to shorten the code